addons/im_livechat/static/src/core/common/chatbot_step_model.js

```diff
@@ -12,9 +12,9 @@ export class ChatbotStep extends Record {
         },
     });
     selectedAnswer = fields.One("chatbot.script.answer");
-    type = fields.Attr("", {
+    step_type = fields.Attr("", {
         compute() {
-            return this.scriptStep?.type;
+            return this.scriptStep?.step_type;
```

Comment:
`this.scriptStep` is a `chatbot.script.step`, not to be confused with `ChatbotStep`.
`chatbot.script.step` still uses `type` rather than `step_type`.

Reply:
Just the attribute not renamed in JS class definition, I made the change.